README.md

```diff
@@ -51,6 +51,7 @@ Then:
 
 Setup and Run the Desktop-based Coldcard simulator:
 
+- `pushd external/micropython/mpy-cross && make && popd`
```

Comment:
Included in README.md for completeness

Reply:
pretty sure "make setup" in ../unix does this already, but haven't checked